### PR TITLE
Turn chronyd module into a short prerequisite

### PR DESCRIPTION
**Pull request by asteflova**
#### What changes are you introducing?

Dropping the module to configure chronyd and replacing it with a short prerequisite.

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

chronyd is installed and enabled by default, a short prerequisite is enough. See also https://issues.redhat.com/browse/SAT-27943

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)

This PR also removes the module on clock sync from Configuring Load Balancing. I don't think that's a problem because the assembly in that guide links to Installing Proxy where the new prerequisite is listed.

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Review by maximiliankolb — APPROVED** [review]
LGTM style-wise

File: guides/common/modules/ref_system-requirements.adoc [hunk 1]
```diff
@@ -94,6 +94,16 @@ ifdef::foreman-el,katello,satellite[]
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
+.Synchronized system clock
+The system clock on the base operating system where you are installing your {ProductName} must be synchronized with Network Time Protocol (NTP) servers.
+If the system clock is not synchronized, SSL certificate verification might fail.
```
[the remark this answers is not in the document]
Comment (evgeni) [reply guides/common/modules/ref_system-requirements.adoc:99]: That depends on the implementation. When users use systemd-timesyncd then yes. Crony? no. ntpd? no.

File: guides/common/modules/ref_system-requirements.adoc [hunk 2]
```diff
@@ -94,6 +94,16 @@ ifdef::foreman-el,katello,satellite[]
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
+.Synchronized system clock
+The system clock on the base operating system where you are installing your {ProductName} must be synchronized with Network Time Protocol (NTP) servers.
```
Comment (evgeni) [inline guides/common/modules/ref_system-requirements.adoc:98]: *technically* there is no requirement that the synchronization happens via NTP (and other protocols exist and are being used).

File: guides/common/modules/ref_system-requirements.adoc [hunk 3]
```diff
@@ -94,6 +94,16 @@ ifdef::foreman-el,katello,satellite[]
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
+.Synchronized system clock
```
Comment (evgeni) [inline guides/common/modules/ref_system-requirements.adoc:97]: this is inside a `ifdef::foreman-el,katello,satellite[]` block, but it applies to *alll* deployments (so also Debian and Orcharino)

Comment (asteflova) [reply guides/common/modules/ref_system-requirements.adoc:98]: Addressed.

File: guides/common/modules/ref_system-requirements.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (asteflova) [reply guides/common/modules/ref_system-requirements.adoc:100]: Ok, let's drop the details about NTP.

File: guides/common/modules/ref_system-requirements.adoc [hunk 4]
```diff
@@ -93,7 +93,19 @@ ifdef::foreman-el,katello,satellite[]
 .SELinux mode
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
+endif::[]
+
+.Synchronized system clock
+The system clock on the base operating system where you are installing your {ProductName} must be synchronized across the network.
+If the system clock is not synchronized, SSL certificate verification might fail.
+ifdef::satellite[]
+For more information, see the following documents:
```
Comment (asteflova) [inline guides/common/modules/ref_system-requirements.adoc]: I also rephrased this to emphasize that chronyd is just one of the services that can be used for timekeeping.

File: guides/common/modules/ref_system-requirements.adoc (hunk 2 above)
Comment (asteflova) [reply guides/common/modules/ref_system-requirements.adoc]: Addressed.

**Review by maximiliankolb — APPROVED** [review]
style-wise still LGTM

**Review by evgeni — APPROVED** [review]

**Pull request merged**